cinema/models.py

```diff
@@ -35,12 +40,21 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_path(
+        instance: (
+            "Movie", "MovieSession"
+        ), filename: str) -> pathlib.Path:
+    _, ext = os.path.splitext(filename)
+    return f"uploads/movies/{slugify(instance.title)}-{uuid.uuid4()}{ext}"
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
     genres = models.ManyToManyField(Genre)
     actors = models.ManyToManyField(Actor)
+    image = models.ImageField(null=True, upload_to="movie_image_path")
```

Comment:
The `upload_to` parameter in the `ImageField` should be a callable (function) or a string path. Since `movie_image_path` is defined as a function, it should not be enclosed in quotes. Remove the quotes to correctly reference the function: `upload_to=movie_image_path`.